cfme/common/provider.py

```diff
@@ -303,8 +310,11 @@ def _fill_default_endpoint_dicts(self, provider_attributes, connection_configs):
             connection_configs.append(default_connection)
         if getattr(endpoint_default, "security_protocol", None):
             security_protocol = endpoint_default.security_protocol.lower()
-            if security_protocol == "basic (ssl)":
+            if security_protocol == "basic (ssl)" or security_protocol == 'ssl without validation':
```

Comment:
This would look/read better as `if security_protocol in ['basic (ssl)', 'ssl without validation']`

Reply:
you're right, I didn't think of optimal code at that moment.